pkg/pod/pods.go

```diff
@@ -971,10 +971,18 @@ func (p *Pod) AppImageManifest(appName string) (*schema.ImageManifest, error) {
 // CreationTime returns the time when the pod was created.
 // This happens at prepare time.
 func (p *Pod) CreationTime() (time.Time, error) {
-	if p.isPrepared || p.isRunning() || p.AfterRun() {
-		return p.getModTime("pod")
+	if !(p.isPrepared || p.isRunning() || p.AfterRun()) {
+		return time.Time{}, nil
+	}
+	t, err := p.getModTime("pod-created")
```

Comment:
Technically, I think flavor is an implementation detail of stage1-coreos/stage1-kvm.
I don't think fly adds it e.g. and it's not referenced in the implementer's guide.

The only other target I thought looked sane was ppid/pid, but creating a new file seemed simpler.